README.md

```diff
@@ -58,6 +63,20 @@ We recommend using [Homebrew](https://brew.sh/), [rbenv](https://github.com/rben
   This command copies sample configuration files, installs required gems
   and sets up the database. Check out our Makefile commands to learn more about what this command does: https://github.com/18F/identity-idp/blob/main/Makefile
 
+If this command errors, you may need to install dependancies outside of the makefile with:
+```
+$ bundle install
+$ yarn install
+```
+
+If setup errors at the mailcatcher installation, ensure you can install the openssl, thin, and mailcatcher gems. For more info, [here is a helpful gist on these errors](https://gist.github.com/pitpit/78fc6b34b1b564c86d80). TL;DR, run:
+
+```
+$ gem install openssl
+$ gem install thin -v 1.5.1 -- --with-cflags="-Wno-error=implicit-function-declaration"
+$ gem install mailcatcher -- --with-cppflags=-I/usr/local/opt/openssl/include
```

Comment:
what if we moved these into the `bin/setup` script so they just work automatically?

Reply:
👍 
I didn't want to change it in case it was just my environment, but if it's a common issue, I'll update that.